openlibrary/macros/AffiliateLinks.html

```diff
@@ -64,4 +63,6 @@
         $for store in more_stores:
           $:affiliate_link(store['key'], store['analytics_key'], store['name'], store['link'], store.get('price', ''), store.get('price_note', ''))
       </details>
+    
+    <small>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</small>
```

Comment:
`small` breaks the styling and I'd rather not add more css for a small semantic victory at this stage. Happy to accept a followup pr to anyone who wants to fix.

```suggestion
    <p>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</p>
```